app/src/shared/errors/ResourceNotFoundError.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+export default class ResourceNotFoundError {
```

Comment:
Didn't make it inherit from `Error` cause it isn't really necessary. Throwing `ResourceNotFoundError` is more of a breakpoint than an "exception" per se.

Reply:
Might be worthwhile extending `Error` anyway just to get the stack trace for future debugging. Remember to `Error.captureStackTrace()` though. Alternatively don't call it an `Error`? `throw new ResourceNotFoundMessage()`?

Reply:
Also thinking maybe it should capture some info about which resource wasn't found? If I'm seeing this and I'm loading multiple resources, which resource didn't load? Will I be able to track it down without a stack?

Reply:
Hm, the point about "Error" in the name does make sense… Will adjust. 🤝